frontend/talentsearch/src/js/components/search/SearchForm.tsx

```diff
@@ -173,8 +174,10 @@ const SearchForm = React.forwardRef<SearchFormRef, SearchFormProps>(
           ...(values.languageAbility !== NullSelection
             ? { languageAbility: values.languageAbility as LanguageAbility }
             : {}), // Ensure null in FormValues is converted to undefined
-          wouldAcceptTemporary:
-            values.employmentDuration === "true" ? true : null,
+          positionDuration:
```

Comment:
Technically this works but this PR is to make it not work like a boolean.  I think it' worth updating the radio selections below to use the new constants rather than continuing to use this like a boolean.  I think the page should send "PERMANENT" to the API, not null, when the permanent radio is selected.

Reply:
changed search/request to send PERMANENT
changed message for no duration selection
https://github.com/GCTC-NTGC/gc-digital-talent/pull/4956/commits/078d061d253a5acaa85792422833caa090b75e14
https://github.com/GCTC-NTGC/gc-digital-talent/pull/4956/commits/65ea0fe52c5f4f11e249baeec3175caa2242c05c